openml_OS/models/api/v1/Api_evaluation.php

```diff
@@ -249,18 +352,7 @@ private function evaluation_list($segs, $user_id, $show_params) {
       $this->returnError(542, $this->version);
       return;
     }
-    
-    if ($show_params) {
```

Comment:
seems that this was unused? thanks for cleaning

Reply:
I am not entirely sure where this change comes from. I merged the dev branch into this to update the current state. Thereby parts of these changes came in. I will dig into that a little more to be on the safe side! (Yet another proof why it is a good thing to have code reviews :) )

Reply:
Okay this part of the code got screwed up during merging the develop branch into this feature branch. I just repaired it.